File: problems/字符串/709.转换成小写字母.py
#!/usr/bin/env python3
"""
709. 转换成小写字母
"""


def to_lower_case(s: str) -> str:
    return s.lower()


if __name__ == "__main__":
    # 测试用例
    print(to_lower_case("Hello"))    # 预期: "hello"
    print(to_lower_case("here"))     # 预期: "here"
    print(to_lower_case("LOVELY"))   # 预期: "lovely"

File: problems/字符串/709.转换成小写字母.cpp
#include <iostream>
#include <string>
using namespace std;

string toLowerCase(string s) {
    for (char& c : s)
        if (c >= 'A' && c <= 'Z')
            c = c - 'A' + 'a';
    return s;
}

int main() {
    // 测试用例
    cout << toLowerCase("Hello") << endl;    // 预期: hello
    cout << toLowerCase("here") << endl;     // 预期: here
    cout << toLowerCase("LOVELY") << endl;   // 预期: lovely
    return 0;
}
